src/components/student/projectqs/ProjectqsForm.js

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { ErrorMessage, Field, Form, Formik } from 'formik';
+import * as Yup from 'yup';
+
+// Render the Formik form
+const renderForm = ({ errors, status, touched, isSubmitting }) => (
+	<Form className="project-quick-start-form">
+		<label>
+			<span className="input-label">Project Title</span>
+			<br />
+			<Field name="project_title" type="text" className="project-title-text-area" />
+			<ErrorMessage name="project_title" component="div" />
+		</label>
+		<label>
+			<span className="input-label">Project Type</span>
+			<br />
+			<Field name="project_type" type="text" className="project-type-text-area"/>
+			<ErrorMessage name="project_type" component="div" />
+		</label>
+
+		<label>
+			<span className="input-label">Live Demo URL</span>
+			<br />
+			<Field name="live_demo_url" type="text" className="live-demo-text-area" />
+			<ErrorMessage name="live_demo_url" component="div" />
+		</label>
+
+        <label>
+			<span className="input-label">Medium Article URL</span>
+			<br />
+			<Field name="medium_article_url" type="text" className="medium-article-text-area" />
+			<ErrorMessage name="medium_article_url" component="div" />
+		</label>
+
+		<label>
+			<span className="input-label">Customer Sales Pitch</span>
+			<br />
+			<Field name="customer_sales_pitch" type="text" className="pitch-text-area" />
+			<ErrorMessage name="customer_sales_pitch" component="div" />
+		</label>
+
+		<label>
+			<span className="input-label">Technical Sales Pitch</span>
+			<br />
+			<Field name="technical_sales_pitch" type="text" className="pitch-text-area" />
+			<ErrorMessage name="technical_sales_pitch" component="div" />
+		</label>
+
+		<button type="submit" disabled={isSubmitting}>
+			Create Project
+		</button>
+	</Form>
+);
+
```

Comment:
It's up to you, but I did separate the Yup schema and the renderForm schema (line 7) into another file for the Create/Edit Profile Form to keep things cleaner. I haven't had a pull request yet that reflects that change, so that's why you likely didn't see that!